docs/atl/reference/ccomcoclass-class.md

```diff
@@ -1,40 +1,39 @@
 ---
-description: "Learn more about: CComCoClass Class"
 title: "CComCoClass Class"
+description: "Learn more about: CComCoClass Class"
 ms.date: "11/04/2016"
 f1_keywords: ["CComCoClass", "ATLCOM/ATL::CComCoClass", "ATLCOM/ATL::CComCoClass::CreateInstance", "ATLCOM/ATL::CComCoClass::Error", "ATLCOM/ATL::CComCoClass::GetObjectCLSID", "ATLCOM/ATL::CComCoClass::GetObjectDescription"]
 helpviewer_keywords: ["CComCoClass class", "aggregation [C++], aggregation models"]
-ms.assetid: 67cfefa4-8df9-47fa-ad58-2d1a1ae25762
 ---
 # CComCoClass Class
 
 This class provides methods for creating instances of a class, and obtaining its properties.
 
 ## Syntax
 
-```
+```cpp
 template <class T, const CLSID* pclsid = &CLSID_NULL>
 class CComCoClass
 ```
 
 #### Parameters
 
-*T*<br/>
+*T*\
 Your class, derived from `CComCoClass`.
 
-*pclsid*<br/>
+*pclsid*\
 A pointer to the CLSID of the object.
 
 ## Members
 
 ### Public Methods
 
-|Name|Description|
-|----------|-----------------|
-|[CComCoClass::CreateInstance](#createinstance)|(Static) Creates an instance of the class and queries for an interface.|
-|[CComCoClass::Error](#error)|(Static) Returns rich error information to the client.|
-|[CComCoClass::GetObjectCLSID](#getobjectclsid)|(Static) Returns the object's class identifier.|
-|[CComCoClass::GetObjectDescription](#getobjectdescription)|(Static) Override to return the object's description.|
+| Name                                                       | Description                                                             |
+| ---------------------------------------------------------- | ----------------------------------------------------------------------- |
+| [CComCoClass::CreateInstance](#createinstance)             | (Static) Creates an instance of the class and queries for an interface. |
+| [CComCoClass::Error](#error)                               | (Static) Returns rich error information to the client.                  |
+| [CComCoClass::GetObjectCLSID](#getobjectclsid)             | (Static) Returns the object's class identifier.                         |
+| [CComCoClass::GetObjectDescription](#getobjectdescription) | (Static) Override to return the object's description.                   |
```

Comment:
Should we format tables as well, to improve readability of the raw markdown source?

Reply:
Good question. I see benefits both ways. I usually don't because it becomes maintenance to keep it that way. As the owner of 10,000 topics I try to reduce maintenance whenever I can.

Reply:
I would think maintenance becomes a factor for tables that update somewhat frequently. Most of these tables never really change in 5+ years, so it might be worth formatting it once and keeping it as such.

Reply:
It depends on the font in your editor whether the cleaned up version looks cleaned up. And we have editor tools that actually do the opposite and remove all the spaces from tables. I see both versions in our docs. I can see the value both ways. But mostly, I'd prefer not to spend time on PRs that clean up tables because its time I can't really afford for a gain that is nebulous in the scheme of things.
I very much appreciate all the other contributions you make because they add clear customer value and are things I'd have to do myself. You've made ongoing and significant improvements to these docs, for which I thank you.

Reply:
Noted! Will not format tables in future PRs since there might not be a clear benefit (with potential downsides).